### PR TITLE
feat: Add pwd command

### PR DESCRIPTION
**Pull request by AhmedEhab2022**
Make pwd command:
- return current working directory
- it works with any argument passed like (pwd cat => <current_working_directory>)

### CODE REVIEW

File: src/interpreter/commands/PwdCommand.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package interpreter.commands;
+
+import interpreter.Command;
+import interpreter.InterpreterContext;
+
+public class PwdCommand {
+    public String execute(Command command, InterpreterContext context) {
+        context.setCurrentDirectory(System.getProperty("user.dir"));
```
Comment (mohanad-80) [inline src/interpreter/commands/PwdCommand.java:8]: Why are you setting the current directory?
The command should return the current directory not update it.
You should only get it and then return it as a string.

File: src/interpreter/CommandHandler.java [hunk 2]
```diff
@@ -29,14 +29,17 @@ public void execute(Command command) {
...
+          break;
         default:
           System.out.println("Unknown command: " + command.getName());
       }
 
       // Handle redirection to a file
       if (command.getOutputFile() != null) {
         try (FileWriter writer = new FileWriter(command.getOutputFile(), command.isAppend())) {
-          writer.write(output);
+          writer.write(output + '\n');
```
Comment (mohanad-80) [inline src/interpreter/CommandHandler.java:42]: Did you test the command with the >> or > operators?

**Review by mohanad-80 — CHANGES_REQUESTED** [review]

File: src/interpreter/CommandHandler.java (hunk 2 above)
Comment (AhmedEhab2022) [reply src/interpreter/CommandHandler.java:42]: Yes, i test it with > and >> in git bash and by default add new line after the content

File: src/interpreter/commands/PwdCommand.java (hunk 1 above)
Comment (AhmedEhab2022) [reply src/interpreter/commands/PwdCommand.java:8]: When i use it it get my home directory as you made in the constructor so i use set property and i searched about it and get this
 https://stackoverflow.com/questions/4871051/how-to-get-the-current-working-directory-in-java

**Pull request closed without merging**